Tests/LeafErrorMiddlewareTests/LeafErrorMiddlewareTests.swift

```diff
@@ -147,6 +157,18 @@ class LeafErrorMiddlewareTests: XCTestCase {
         XCTAssertEqual(contextDictionary["status"], "401")
         XCTAssertEqual(contextDictionary["statusMessage"], "Unauthorized")
     }
+    
+    func testThatFuture404IsCaughtCorrectly() throws {
+        let response = try app.getResponse(to: "/future404")
+        XCTAssertEqual(response.http.status, .notFound)
+        XCTAssertEqual(viewRenderer.leafPath, "404")
+    }
+    
+    func testThatFuture403IsCaughtCorrectly() throws {
+        let response = try app.getResponse(to: "/future403")
+        XCTAssertEqual(response.http.status, .internalServerError)
```

Comment:
This should come back as a `.forbidden` error, instead of an internal server error. Instead of looking at the localised description (which is liable to change), it's probably better to try and cast the caught error to an `Abort` error then use the status code from that, only falling back to an internal server error if the cast fails